conf/config.level0.neon

```diff
@@ -56,6 +56,7 @@ rules:
 	- PHPStan\Rules\Classes\NonClassAttributeClassRule
 	- PHPStan\Rules\Classes\TraitAttributeClassRule
 	- PHPStan\Rules\Constants\FinalConstantRule
+	- PHPStan\Rules\Constants\MagicClassConstantRule
```

Comment:
This should be enabled only with bleeding edge. The code does not error, it just returns an empty string AFAIK.

Reply:
I have used a feature flag with a more general name, so I can use it for similar rules for `__FUNCTION__` and `__METHOD__`